practice/intlist.c

```diff
@@ -0,0 +1,172 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// intlist.c
+// Program for Linked List functions.
+// ------------------------------------------------------------------------------------------------
+#include <stdio.h>
+#include <stdlib.h>
+#include <malloc.h>
+#include "intlist.h"
+// Function to create a new linked list
+LinkedList* Create () {
+	LinkedList* list = (LinkedList*)malloc (sizeof (LinkedList)); // Allocate memory for the list structure
+	if (list == NULL) {
+		printf ("Memory allocation failed\n");

```

Comment:
Library functions do not output messages. They communicate with the calling functions only through error codes.